### ISSUE TITLE
Request: Add BiomeProviderList

### ISSUE THREAD

**Issue by zauberparacelsus** [maruohon/justenoughdimensions#49]
I was looking at this mod and hoping it could be used as a way of creating a custom dimension with a limited set of biomes, but that doesn't appear to be the case, since the only option for specifying biomes is to use BiomeProviderSingle.

I'd like to propose a new biome provider, BiomeProviderList.  It differs in that instead of providing a string containing a single biome, it would use a JSON list, with each entry in the list being a string with a single biome.

**Comment by maruohon** [comment]
This is the number one thing I would like to add, and which gets asked about quite regularly.

But it would also require a somewhat significant time investment to learn how to actually implement, as currently I don't know how the noise field stuff works that is used for biome generation (among other things). Also in the past almost two years I've mostly switched into working on my client-side mods, so JED has gotten a lot less development time in general lately.

So basically I have no idea when I'll end up looking into this...
